council/llm/llm_function/llm_function.py

```diff
@@ -199,7 +202,7 @@ def execute_with_llm_response(
         """
 
         llm_messages: List[LLMMessage] = self._messages + self._validate_messages(
-            user_message, messages, LLMMessageRole.User
+            user_message, messages, LLMMessageRole.User, allow_empty_input=True
         )
```

Comment:
Do we want to `RuntimeError` if `len(llm_messages) == 0`?

Reply:
`llm_messages` will not be empty because `allow_empty_input=False` for `self._messages` in `__init__` and it's raise a `ValueError`